release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -539,6 +541,20 @@ For more information, see xref:../networking/metallb/about-metallb.adoc#about-me
 [id="ocp-4-10-builds"]
 === Builds
 
+* With this update, you can run entitled builds with `SharedSecret` objects as a Technology Preview feature. This feature relies on the newly-introduced a new Shared Resource driver and the Insights Operator to import {op-system-base} Simple Content Access (SCA) certificates. By using this feature, you can install entitled RPM packages during builds without the extra effort of copying your {op-system-base} subscription credentials and certificates into the builds' namespaces. (link:https://issues.redhat.com/browse/BUILD-274[BUILD-274])
++
+[IMPORTANT]
+====
+The `SharedSecret` objects and OpenShift Shared Resources feature are only available if you enable the `TechPreviewNoUpgrade` feature set. These Technology Preview features are not part of the default features. Enabling this feature set cannot be undone and prevents upgrades. This feature set is not recommended on production clusters. See xref:../post_installation_configuration/cluster-tasks.adoc#post-install-tp-tasks[Enabling Technology Preview features using FeatureGates].
+====
+
+* With this update, workloads can securely share `Secrets` and `ConfigMaps` objects across namespaces using inline ephemeral `csi` volumes provided by the Shared Resource CSI Driver. Container Storage Interface (CSI) volumes and the Shared Resource CSI Driver are Technology Preview features. (link:https://issues.redhat.com/browse/BUILD-293[BUILD-293])
+
+[id="ocp-4-10-jenkins"]
+=== Jenkins
+
+* With this update, you can run the Jenkins agent as a sidecar container. You can use this to run any container image in a Jenkins pipeline with a correctly-configured pod template and Jenkinsfile. (link:https://issues.redhat.com/browse/JKNS-132[JKNS-132])
```

Comment:
```suggestion
* With this update, you can run the Jenkins agent as a sidecar container. You can use this to run any container image in a Jenkins pipeline with a correctly configured pod template and Jenkinsfile. (link:https://issues.redhat.com/browse/JKNS-132[JKNS-132])
```
- Per ISG section on compound words, "Do not use a hyphen after an adverb or adjective that ends in -ly because such terms are never ambiguous."

Reply:
Updated. Thanks.